public/js/qr-creator.js

```diff
@@ -15,7 +15,7 @@
       var g = a, x = c.left + b * h, u = c.top + r * h, p = r, q = b, f = d.a, k = x + h, m = u + h, D = p - 1, E = p + 1, n = q - 1, l = q + 1, y = Math.floor(Math.min(.5, Math.max(0, c.R)) * h), v = f(p, q), I = f(D, n), w = f(D, q); D = f(D, l); var F = f(p, l); l = f(E, l); q = f(E,
         q); E = f(E, n); p = f(p, n); x = Math.round(x); u = Math.round(u); k = Math.round(k); m = Math.round(m); v ? C(g, x, u, k, m, y, !w && !p, !w && !F, !q && !F, !q && !p) : z(g, x, u, k, m, y, w && p && I, w && F && D, q && F && l, q && p && E)
     } A(a, c); a.fill(); return t
-  } var v = { minVersion: 1, maxVersion: 40, ecLevel: "L", left: 0, top: 0, size: 200, fill: "#000", background: null, text: "no text", radius: .5, quiet: 0 }; G = function(t, c) {
+  } var v = { minVersion: 1, maxVersion: 40, ecLevel: "L", left: 0, top: 0, size: 200, fill: (window.matchMedia('(prefers-color-scheme: dark)').matches ? "#FFF" : "000"), background: null, text: "no text", radius: .5, quiet: 0 }; G = function(t, c) {
```

Comment:
I am hesitant making a change in here because it is an external library and not our code. 
Do we need to make this change? maybe it's ok to keep the QR code background white?

Reply:
It looks like background and fill can be passed to the library in the render method https://larsjung.de/jquery-qrcode/

Reply:
Ah thanks didn't see that!